### DISCUSSION TITLE
After Monterey upgrade: 100% Brightness is actual 50%

### DISCUSSION THREAD

**Discussion by mattefabb** [MonitorControl/MonitorControl#912] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Looked through [the wiki](https://github.com/MonitorControl/MonitorControl/wiki)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the bug

MonitorControl perfectly worked until the Monterey OS upgrade: the second after the Brightness was real 100% of the external monito (Apple Cinema LED), but when I tried to scroll the brightness > it went 50% and is still at it (MonitorControl think it's 100%).
I'm on a M1 Macbook Pro.

### Steps to reproduce

1 Upgrade to Monterey
2 Try to adjust Brightness
3 Try to bring it back to 100%

### Expected behavior

Expected to come back to 100%, at 100% of setting

### Environment Information (please complete the following information)

```markdown
- macOS version: Monterey
- Mac model: M1
- MonitorControl version: Latest
- Monitor(s): Apple Cinema LED
- Apple Silicon/M1 (yes or no): yes
```

**Comment by mattefabb** [comment]
Update: Lunar can do the trick, in addition of MonitorControl I installed Lunar pro (trial) and it was able to bring bag the Brightness to real 100%, but MonitorControl keep doing the bug: if I try to adjust its 100% Brightness bring back the monitor to ist 50% actual Brightness and I need Lunar to bring it back to 100%... I hope MonitorControl will fix it, or I will be forced to purchase Lunar Pro for the future.

**Comment by the0neyouseek** [comment]
@mattefabb Please have a look here https://github.com/MonitorControl/MonitorControl/discussions/739 and tell me if it fixes your issue

**Comment by waydabber** [comment]
We should investigate this issue but a bit difficult without an actual display like this. I guess we should wait for somebody with the expertise and the hardware hands-on to figure out what is wrong.

Btw, Lunar is a great product I'd say go for it if it fixes the issue for you! @alin23 :)

**Comment by DDCHlsq** [comment]
I have the exactly same issue. My external display does not go any brighter after brightness control hits 50%.
```
OS: MacOS Monterey 12.2.1
Model: MacBook Pro 14-inch (M1 Pro)
Display Model: AOC LV273HUPR
MonitorControl Version: 4.0.2 (6965)
```

**Reply by the0neyouseek** [reply]
Hi, @DDCHlsq 
Please have a look here first https://github.com/MonitorControl/MonitorControl/discussions/739 and tell me if it fixes your issue.

**Reply by DDCHlsq** [reply]
Hi @the0neyouseek , I've tried setting the max value of DDC to 250 but it doesn't help. Should I do anything else to fix it
<img width="842" alt="截屏2022-02-18 22 58 40" src="https://user-images.githubusercontent.com/49261063/154707051-add3fd53-b219-45f3-9549-863b5959dc5c.png">
?

**Reply by the0neyouseek** [reply]
It seems to be a display issue, see if you can control your display with ddc using a tool like https://github.com/waydabber/m1ddc 
If not, you can disable DDC control by unchecking this 
![SCR-20220218-nz4](https://user-images.githubusercontent.com/4009175/154720718-400efd60-8035-4f92-b9b2-f72541ca6d13.png) 
and MonitorControl will use gamma control to change brightness levels

**Reply by DDCHlsq** [reply]
Hi @the0neyouseek 
Thank you for your help. I compiled the tool you provided and got an I2C communication failure trying using it.
Later I checked the OSD settings of my display and found that I didn't turn on the DDC/CI switch. After toggling the switch, MonitorControl works perfectly fine via DDC ;)

**Reply by the0neyouseek** [reply]
@DDCHlsq Great! Glad I could help!
